pkg/controllers/management/auth/project_cluster_handler.go

```diff
@@ -69,10 +70,14 @@ type projectLifecycle struct {
 }
 
 func (l *projectLifecycle) sync(key string, orig *v3.Project) (runtime.Object, error) {
-	if orig == nil {
+	if orig == nil || orig.DeletionTimestamp != nil {
 		return nil, nil
 	}
 
+	if err := l.enqueueCrtbs(orig); err != nil {
```

Comment:
do we want this to run before reconcileNamespace? Probably won't lead to an error but if there ends up being an error during reconcile we probably didn't need to enqueue, since the ns for it will not exist yet.

Reply:
makes sense, moved it